.github/update-sha.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Script to update the SHA version
+
+# get name and version from dependabots last commit message
+last_commit=$(git log --author=dependabot --pretty=oneline -1)
+
+if [[ $last_commit =~ '.*Bump.*from.*to.*' ]]
+then
+    name=$(echo $last_commit | grep -o -P '(?<=Bump ).*(?= from)')
+    old_version=$(echo $last_commit | grep -o -P '(?<=from ).*(?= to)')
+    new_version=$(echo $last_commit | grep -o -P '(?<=to ).*(?= in)')
+
+    # find corresponding package file
+    # CI executes at the root of the repository
+    cd ./packages/
+    directory=$(find . -maxdepth 1 -name '*'$name'*' -print)
+    cd $directory
+
+    # substitute new url of updated version
+    old_url_line=$(cat package.py | grep "url.*=.*${old_version}.*")
```

Comment:
```suggestion
    old_url=$(cat ./package.py | grep 'url * = *"https' -m 1 | grep -oP '"\K[^"\047]+(?=["\047])')
    old_file=$(basename -- ${old_url})
    old_version=$(echo ${old_file} | sed -e "s/.bz2//;s/.dmg//;s/.tgz//;s/.gz//;s/.zip//;s/.xz//;s/.tar//;")
    old_url_line=$(cat package.py | grep "url.*=.*${old_version}.*")
```

Pull old url out from package.py, get the old filename from the old url, remove common extensions to get just the version number